api/cpp/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
 
 cmake_minimum_required(VERSION 3.21)
-project(Slint HOMEPAGE_URL "https://slint-ui.com/" LANGUAGES CXX)
+project(Slint HOMEPAGE_URL "https://slint-ui.com/" LANGUAGES CXX VERSION 1.0.0)
```

Comment:
I thought we would need to add a regexp in the upgrade_version.yaml script, but this is already there it seems.